triton/tut/gpu.rst

```diff
@@ -312,19 +312,231 @@ on efficient data loading.
 
 .. _cuda-nvprof:
 
-Profiling GPU usage with nvprof
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Profiling GPU usage with NVIDIA Profilers
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
The section header level could be increased so that this section appears on the left side bar. So change of header character from `~` to `-`.